src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -165,6 +165,7 @@ export interface ITerminalConfiguration {
 	scrollback: number;
 	commandsToSkipShell: string[];
 	allowChords: boolean;
+	allowSmartPaste: boolean;
```

Comment:
We want this to live entirely within `terminalContrib/clipboard`, including the configuration setup